.github/workflows/builddoc.yml

```diff
@@ -14,30 +14,21 @@ concurrency:
 
 env:
   FORCE_COLOR: "1"
-  UV_SYSTEM_PYTHON: "1"  # make uv do global installs
+  UV_VERSION: 0.3.2  # Pin the version of UV
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v4
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: "3"
     - name: Install graphviz
       run: sudo apt-get install graphviz
     - name: Install uv
-      run: >
-        curl --no-progress-meter --location --fail
-        --proto '=https' --tlsv1.2
-        "https://astral.sh/uv/install.sh"
-        | sh
-    - name: Install dependencies
-      run: uv pip install .[docs]
+      run: curl -LsSf https://astral.sh/uv/${{ env.UV_VERSION }}/install.sh | sh
```

Comment:
If possible: could we use the [`astral-sh/setup-uv` GitHub Action](https://github.com/astral-sh/setup-uv) here, instead of installing from a hosted scriptfile?

Reply:
Ah: I notice we're already using the scriptfile version in a few workflows.  Ignore me; I'll suggest that we migrate them collectively.